CHANGELOG

```diff
@@ -1,6 +1,10 @@
 devel
 -----
 
+* Whenever there is an query request timeout in AQL (e.g. a server died)
+  which causes the query to fail, the DBServers will now all directly kill
+  their potentially ongoing parts of the query and not wait for garbage collection.
```

Comment:
Strictly speaking we were not just waiting for the garbage collection to clean up, but the query would run until completion and only _then_ would it be garbage collected.

Reply:
Agreed, however I think this is good enough for a changelog. The important thing here is -> Abort != Free resources.